### PR TITLE
Adding Solid World Summary

### PR DESCRIPTION
**Pull request by MarrelleBailey**

### CODE REVIEW

**Review by justinwb — CHANGES_REQUESTED** [review]
@MarrelleBailey No issues with the text but there should be a bit of a lead-in with the date and event name (see the event after it that starts with data and title of event).

File: _posts/newsletter/next.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ categories: [Updates]
 
 ### [Events](https://solidproject.org/events)
 
+This month at Solid World we heard from a pair of guest presenters diving into the practicalities of building sophisticated, scalable Solid apps and services.
```
Comment (Vinnl) [inline _posts/newsletter/next.md:14]:
```suggestion
1 October 2020 - This month at Solid World we heard from a pair of guest presenters diving into the practicalities of building sophisticated, scalable Solid apps and services.
```

File: _posts/newsletter/next.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ categories: [Updates]
...
+
+We also heard from one of Inrupt’s engineering leads, Jack Lawson, about their React SDK V2. To begin, he went over the three major components that make up the React SDK — authentication, visualization, and data syncing. Next, he discussed how the open source toolkit is meant to make it easy for developers to build Solid applications, avoiding stress over tedious details, offering a smooth implementation, and providing the opportunity to leverage Inrupt’s best practices and coding standards. Unfortunately, we ran out of time to see the demo, but plan to see in the near future!   
+
+If you missed Solid World you can look out for recordings of the guest presentations coming soon to [our Vimeo page](https://vimeo.com/solidworld). The next Solid World will take place on the 5th November 2020. Follow along at [solidproject.org/events](http://solidproject.org/events) as we finalize the agenda, which will highlight Solid-based projects from several industries and research areas.
```
Comment (Vinnl) [inline _posts/newsletter/next.md:22]:
```suggestion
If you missed Solid World you can look out for recordings of the guest presentations coming soon to [our Vimeo page](https://vimeo.com/solidworld). The next Solid World will take place on 5 November 2020. Follow along at [solidproject.org/events](https://solidproject.org/events) as we finalize the agenda, which will highlight Solid-based projects from several industries and research areas.
```

**Review by Vinnl — APPROVED** [review]

**Comment by Vinnl** [comment]
Thanks!

**Pull request merged**